nikola/filters.py

```diff
@@ -45,6 +45,19 @@
 from .utils import req_missing, LOGGER
 
 
+class configurable_filter(object):
```

Comment:
A better name would be `_ConfigurableFilter`, because we don’t want this to be considered public.